assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -81,10 +88,24 @@ const EditorWizardModal = () => {
 		} );
 	};
 
+	// TODO Implement different flows depending on post type 👇.
+	const steps = [ CourseDetailsStep, UpgradeStep, CoursePatternsStep ];
```

Comment:
I really liked this.

Initially, I was thinking about implementing something the component's `children` (or maybe even React Contexts) to get the steps correctly, but that would definitely lead to a more complex implementation without a clear benefit.

I was a little worried about passing data through from the step to the other components, but I feel like the `setData` should be quite enough. I really liked your approach here. :D

Reply:
Thanks for the good words @fjorgemota. I am happy that we both followed more or less the same steps. I initially tried with `children` and then started playing around passing the context but as you said... I couldn't see the benefit and it was more complex (at least for my understanding).

🙌